python/packages/autogen-agentchat/src/autogen_agentchat/messages.py

```diff
@@ -134,7 +134,8 @@ class MemoryQueryEvent(BaseAgentEvent):
 class ModelClientStreamingChunkEvent(BaseAgentEvent):
     """An event signaling a text output chunk from a model client in streaming mode."""
 
-    content: str
+    content: str | None = None
+    reasoning: str | None = None
```

Comment:
As explained in a different place. I think we don't need to add the `reasoning` field.